lib/matplotlib/tests/test_quiver.py

```diff
@@ -74,6 +74,20 @@ def test_quiver_with_key():
                                       'size': 'large'})
 
 
+@image_comparison(baseline_images=['quiver_orientation_test_image'],
```

Comment:
True, but it will also make it very difficult to interpret the plot if someone wants to understand the behavior it is testing.